test/core/libp2p/muxer/yamux/yamux_frame_test.cpp

```diff
@@ -23,7 +23,7 @@ class YamuxFrameTest : public ::testing::Test {
    */
   void checkFrame(std::optional<YamuxFrame> frame_opt, uint8_t version,
                   YamuxFrame::FrameType type, YamuxFrame::Flag flag,
-                  Yamux::StreamId stream_id, uint32_t length,
+                  YamuxedConnection::StreamId stream_id, uint32_t length,
                   const Buffer &frame_data) {
     ASSERT_TRUE(frame_opt);
```

Comment:
What is the meaning of optional if it must always be not empty? Maybe change design?

Reply:
This argument is a result of `parseFrame` function. We don't want to unwrap the optional every time in the tests, so this was delegated to the `checkFrame`